coalib/bearlib/abstractions/Linter.py

```diff
@@ -196,6 +197,11 @@ def check_prerequisites(cls):
                         check_call(options["prerequisite_check_command"],
                                    stdout=DEVNULL,
                                    stderr=DEVNULL)
+                        warnings.simplefilter("default")
```

Comment:
This doesn't lacks the fact that we need a warning that can be seen, otherwise nobody recognizes and as soon as we remove the deprecated feature, things are going to fail. Better a warning than an error long-term.